signalfx-org-metrics/metrics.yaml

```diff
@@ -1115,7 +1115,7 @@ sf.org.numLimitedMetricTimeSeriesCreateCallsByToken:
   description: |
     One value per metric type per token, each representing the number
     of metric time series (MTS) not sent to Infrastructure Monitoring because you've reached your
-    maximum active MTS allowed.
+    maximum limit for active MTS, hosts, containers, custom metrics, or datapoints per minute (DPM).
```

Comment:
There are a few other things that need to be included here:
"active MTS, host metrics, container metrics, custom metrics, bundled metrics, function metrics, or high resolution metrics"

Reply:
Updated - thank you!